### PR TITLE
Added inputs and flag(reversed) and also two useState

### PR DESCRIPTION
**Pull request by MathewWith**
Дорогой Руслан Андреевич, проверьте пожалуйста и укажите на ошибки, всё исправлю, проблема не будет

### CODE REVIEW

**Comment by CodingRuslan** [comment]
Dear Ruslan Albertovich...

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 import { getAllCurrencies } from 'src/helpers/getAllCurrencies';
-import { ListCurrencies } from 'src/helpers/list-currencies/List-currencies';
+import { ListCurrencies } from 'src/helpers/List-currencies/List-currencies';
 
 function App() {
+  
   const [allCurrencies, setAllCurrencies] = useState([])
+  const [firstInputValue, setFirstInputValue] = useState('')
+  const [secondInputValue, setSecondInputValue] = useState('')
+  const [reversed, setReversed] = useState(false)
```
Comment (CodingRuslan) [inline src/App.tsx]: Flag should have **is**(isReversed)

File: src/App.tsx [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { getAllCurrencies } from 'src/helpers/getAllCurrencies';
-import { ListCurrencies } from 'src/helpers/list-currencies/List-currencies';
+import { ListCurrencies } from 'src/helpers/List-currencies/List-currencies';
 
 function App() {
+  
   const [allCurrencies, setAllCurrencies] = useState([])
+  const [firstInputValue, setFirstInputValue] = useState('')
```
Comment (CodingRuslan) [inline src/App.tsx]: You have left and right inputs, use right naming for variables

File: src/App.tsx [hunk 3]
```diff
@@ -14,6 +18,10 @@ function App() {
     <div className="App">
       <h1>Change money</h1>
       {ListCurrencies(allCurrencies)}
+      <div>
+          <input type="text" onClick={() => setReversed(prev => !prev)} onChange={(e) => setFirstInputValue(e.target.value)}/>
```
Comment (CodingRuslan) [inline src/App.tsx]: What happens if I will clicks only for first input ?

Comment (MathewWith) [reply src/App.tsx]: You will change the flag(isReversed)
*click = true
*click = false 
Yes, if user will click two times on the same input that our converter will not work correctly(if we look to the future)

Comment (CodingRuslan) [reply src/App.tsx]: fix it

**Pull request merged**